src/Client.php

```diff
@@ -433,8 +433,8 @@ protected function createPipeline(array $options = null, $callable = null)
             $class = 'Predis\Pipeline\Pipeline';
         }
 
-        /*
-         * @var ClientContextInterface
+        /**
```

Comment:
This change is important. `/*` is VERY different from `/**`. PHP even tokenizes it differently.